src/app/qgsrasterlayerproperties.h

```diff
@@ -206,7 +206,7 @@ class APP_EXPORT QgsRasterLayerProperties : public QgsOptionsDialogBase, private
     qreal mGradientWidth;
 
     QgsMapCanvas *mMapCanvas = nullptr;
-    QgsMapToolEmitPoint *mPixelSelectorTool = nullptr;
+    std::unique_ptr<QgsMapToolEmitPoint> mPixelSelectorTool = nullptr;
```

Comment:
I'm not sure that a *unique_ptr* needs to be initialized with a `nullptr`.

Reply:
yes, it should be empty by default (that is the same as being initialized with a nullptr).